lab_4_retrieval_w_clustering/main.py

```diff
@@ -81,6 +97,20 @@ def _calculate_bm25(self, tokenized_document: list[str]) -> Vector:
         Returns:
             Vector: BM25 vector for document.
         """
+        if not isinstance(tokenized_document, list) or len(tokenized_document) == 0:
+            raise ValueError
+        vector = [0.0 for elem in range(len(self._vocabulary))]
```

Comment:
try to use another way to get a list of a certain length with the same values, remember the seminar